src/Elasticsearch/Endpoints/Indices/Reindex.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace Elasticsearch\Endpoints\Indices;
```

Comment:
We need to move this to the "main" namespace because the API is just `/_reindex`.  So just move this file to `src/Elasticsearch/Endpoints/Reindex.php` and modify the namespace accordingly.